src/assets/AddTimeCardPeriod.scss

```diff
@@ -0,0 +1,4 @@
+.add-timecard-period-type {
+  border: 1px solid #b1b4b6;
+  padding: 15px;
+}
```

Comment:
In the PR for eahw-1348, we renamed this to `.grey-border` and put it in a `Timecard.scss` file so that other files could use it, depending on if our PR gets merged first you could use that instead

Reply:
I have mimicked your corresponding change in this PR: https://github.com/UKHomeOffice/callisto-ui/pull/18

Reply:
Would we be able to move this into `Timecard.scss` and delete the `AddTimecardPeriod.scss` file?

Reply:
I could move it to `Timecard.scss` but that would be duplicating what you added in `common.scss` in PR #18 
I can't delete `AddTimecardPeriod.scss` as it has the following style:
`.add-timecard-period-type-description {
  color: $govuk-secondary-text-colour;
}`
unless I can refer to`govuk-secondary-text-colour` color directly in the html template ?

Reply:
Sorry, my fault - I meant the `.add-timecard-period-type-description` to move to `Timecard.scss` or `common.scss` depending on reusability and rename it to something generic - I think @daleelR has left a similar comment so I'll resolve this one

Reply:
No problem. How about referring to the secondary text colour directly in the HTML template?
See my latest push